pyquil/device/_specs.py

```diff
@@ -135,6 +135,26 @@ def fCZs(self):
         """
         return {tuple(es.targets): es.fCZ for es in self.edges_specs}
 
+    def fISWAPs(self):
+        """
+        Get a dictionary of ISWAP fidelities (normalized to unity) from the specs,
+        keyed by targets (qubit-qubit pairs).
+
+        :return: A dictionary of ISWAP fidelities, normalized to unity.
+        :rtype: Dict[tuple(int, int), float]
+        """
+        return {tuple(es.targets): es.fISWAP for es in self.edges_specs}
+
+    def fXYs(self):
+        """
+        Get a dictionary of XY/2 fidelities (normalized to unity) from the specs,
```

Comment:
did you mean XY or XY/2 here?

Reply:
I did mean XY/2. This breaks convention, but I found `fXYhalvess` upsetting to look at.

Reply:
the fidelities we'll be storing for pyquil to pull from will (at least for now) be the fidelity of the full XY though (probably XY(pi))

Reply:
oh, well, i find that uncomfortable. is there a strong reason not to store fidelities that reflect the exact gates we're exposing?

Reply:
I thought for now we were exposing only full XYs, and just leaving the possibility of XY/2 open?

Reply:
Oh: my understanding was that we were exposing either XY(pi/2) or "the full XY family", but not XY(pi) = ISWAP. Are we intending only to expose ISWAP?

Reply:
pursuant to this, i've dropped `XYhalves` for now. when we're ready to expose it, we can add it.

Reply:
still need to update this docstring?

Reply:
good catch